snap/quota/quota.go

```diff
@@ -271,13 +273,30 @@ func (grp *Group) JournalNamespaceName() string {
 	return fmt.Sprintf("snap-%s", grp.Name)
 }
 
-// JournalFileName returns the name of the journal configuration file that should
+// JournalConfFileName returns the name of the journal configuration file that should
 // be used for this quota group. As an example, a group named "foo" will return a name
 // of journald@snap-foo.conf
-func (grp *Group) JournalFileName() string {
+func (grp *Group) JournalConfFileName() string {
 	return fmt.Sprintf("journald@%s.conf", grp.JournalNamespaceName())
 }
 
+// JournalServiceName returns the systemd service name for the quota group.
+func (grp *Group) JournalServiceName() string {
+	return fmt.Sprintf("systemd-journald@%s.service", grp.JournalNamespaceName())
```

Comment:
Sorry, I meant %I. But I see that it is not important for `systemd-journald@.service`. So it is OK, as long as we verify somewhere that this is valid instance name.

Reply:
This was a really nice point - I'll do this in a followup since it turned out to involve some more changes than I feel comfortable to do in this PR. 

Fortunately enough we have very strict validation for group names, so we don't allow any special character anyway, but I still feel like it's something we should do as we already do escape group names for slice namings.

Added this to my TODO!